src/MoBi.Presentation/Tasks/Interaction/InteractionTask.cs

```diff
@@ -124,5 +127,17 @@ private void updateFormulaCacheOfClone(IBuildingBlock originBuildingBlock, IBuil
       public bool CorrectName<T>(T objectBase, IEnumerable<string> forbiddenNames) where T : IObjectBase => _nameCorrector.CorrectName(forbiddenNames, objectBase);
 
       public string PromptForNewName<T>(T objectBase, IEnumerable<string> forbiddenNames) where T : IObjectBase => _nameCorrector.PromptForCorrectName(forbiddenNames.ToList(), objectBase);
+
+      public void SaveMultiple<T>(IReadOnlyList<T> entitiesToSerialize)
```

Comment:
Yes we could, but every time we consume this we will have to change the caller, creating a list and adding whatever object into it.
This seems to be easy since this "Save" function now has only 2 references, but this is being called from here:
EditTaskFor.cs:
![image](https://github.com/user-attachments/assets/9444afce-ce3a-4a07-bdd0-99fb40b54e06)

Which as you can see on the image, it has 7 references.
So shall I go for it and change all the 7 callers as well? 
I`d rather not since it will create a lot of changes outside this scope. But let me know, happy to do it if you think it should be the way

Reply:
Yeah, you could just override in InteractionTask, then you only have to create the enumerable in EditTaskFor, so don't change the signature in EditTaskFor, but do change in InteractionTask.